src/main/scala/nl/vroste/zio/kinesis/client/DefaultClientsLayers.scala

```diff
@@ -0,0 +1,31 @@
+package nl.vroste.zio.kinesis.client
+
+import software.amazon.awssdk.services.cloudwatch.CloudWatchAsyncClient
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
+import software.amazon.awssdk.services.kinesis.KinesisAsyncClient
+import zio.{ Has, ZIO, ZLayer, ZManaged }
+
+object DefaultClientsLayers {
+
+  def kinesisAsyncClientLayer: ZLayer[Any, Throwable, Has[KinesisAsyncClient]] =
+    ZLayer.fromManaged(ZManaged.fromAutoCloseable {
```

Comment:
I think there's also `.toLayer` on `ZManaged`

Reply:
I tried this but it lead to compile errors due to type inference problems